builds/jazz-build-module/config-loader.groovy

```diff
@@ -31,4 +31,14 @@ def parseJson(def json) {
     new groovy.json.JsonSlurperClassic().parseText(json)
 }
 
+def getLambdaServiceList() {
+    def service_list = ["cloud-logs-streamer", "deployments-event-handler", "environment-event-handler"]
+    return service_list
+}
+
+def getApiServiceList() {
+    def service_list = ["acl", "create-serverless-service", "environment-event-handler", "delete-serverless-service"]
```

Comment:
These should be pulled from jazz configuration db/installer vars during build time. Installer should add these to configuration during install time.

Reply:
Done